var/spack/repos/builtin/packages/py-recommonmark/package.py

```diff
@@ -19,6 +19,6 @@ class PyRecommonmark(PythonPackage):
 
     version('0.6.0', sha256='29cd4faeb6c5268c633634f2d69aef9431e0f4d347f90659fd0aab20e541efeb')
 
-    depends_on('py-commonmark@0.8.1:')
-    depends_on('py-docutils@0.11:')
-    depends_on('py-sphinx@1.3.1:')
+    depends_on('py-commonmark@0.8.1:', type=('build', 'run'))
+    depends_on('py-docutils@0.11:', type=('build', 'run'))
+    depends_on('py-sphinx@1.3.1:', type=('build', 'run'))
```

Comment:
Missing a build dep on setuptools. Missing a build/run dep on future. I don't actually see any deps on docutils or sphinx, those are probably only needed by developers. Also, the dependency on itself doesn't make a ton of sense 😄

Reply:
Thanks!
Setuptools & future: will add.
sphinx/docutils: yeah, weird because it's a plugin but already declared as such here: https://github.com/readthedocs/recommonmark/blob/master/setup.py#L25-L29 will be used for sphinx builds in most cases anyway...
Not a self-dependency: `py-recommonmark` seems to depend on `py-commonmark` (note the missing `re` :) )

Reply:
Ah, sorry for all of the confusion. I didn't realize there were two packages here and I was looking at the tarball from `commonmark`. The setuptools dep was missing, but the future dep isn't needed by `recommonmark`, only by `commonmark`. I think if you remove future we'll be good to go.